content/docs/create-a-new-react-app.md

```diff
@@ -1,94 +1,94 @@
 
 ### Next.js {#nextjs}
 
-[Next.js](https://nextjs.org/) is a popular and lightweight framework for **static and server‑rendered applications** built with React. It includes **styling and routing solutions** out of the box, and assumes that you're using [Node.js](https://nodejs.org/) as the server environment.
+[Next.js](https://nextjs.org/)<sub>`eng`</sub>-ը հայտնի և թեթև framework է, React-ով ստեղծված, **ստատիկ և սերվերում արտապատկերվող հավելվածների** համար։ Այն ներառում է **ձևավորման և routing-ի լուծումներ** և ենթադրում է, որ դուք օգտագործում եք [Node.js](https://nodejs.org/)<sub>`eng`</sub>-ը որպես սերվերի միջավայր։
 
-Learn Next.js from [its official guide](https://nextjs.org/learn/).
+Սովորիր Next.js-ը նրա պաշտոնական [ձեռնարկից](https://nextjs.org/learn/)<sub>`eng`</sub>։
 
 ### Gatsby {#gatsby}
 
-[Gatsby](https://www.gatsbyjs.org/) is the best way to create **static websites** with React. It lets you use React components, but outputs pre-rendered HTML and CSS to guarantee the fastest load time.
+[Gatsby](https://www.gatsbyjs.org/)<sub>`eng`</sub>-ն լավագույն տարբերակն է React-ով **ստատիկ վեբկայքեր** ստեղծելու համար։ Դա թույլ է տալիս ձեզ օգտագործել React կոմպոնենտներ, բայց ելքային արժեքում ստանալ նախա-արտապատկերված HTML և CSS\` երաշխավորելու համար ամենաարագ ներբեռնումը։
 
-Learn Gatsby from [its official guide](https://www.gatsbyjs.org/docs/) and a [gallery of starter kits](https://www.gatsbyjs.org/docs/gatsby-starters/).
+Սովորիր Gatsby-ն նրա պաշտոնական [ձեռնարկից](https://www.gatsbyjs.org/docs/)<sub>`eng`</sub> և [այս մեկնարկային փաթեթների ցանկից](https://www.gatsbyjs.org/docs/gatsby-starters/)։
 
-### More Flexible Toolchains {#more-flexible-toolchains}
+### Ավելի ճկուն գործիքակազմեր {#more-flexible-toolchains}
 
-The following toolchains offer more flexiblity and choice. We recommend them to more experienced users:
+Ներկայացված գործիքակազմերը առաջարկում են ավելի մեծ ճկունություն և ընտրություն։ Վերջիններս մենք առաջարկում ենք ավելի փորձառու օգտագործողներին։
 
-- **[Neutrino](https://neutrinojs.org/)** combines the power of [webpack](https://webpack.js.org/) with the simplicity of presets, and includes a preset for [React apps](https://neutrinojs.org/packages/react/) and [React components](https://neutrinojs.org/packages/react-components/).
+- **[Neutrino](https://neutrinojs.org/)**<sub>`eng`</sub>-ն համատեղում է իր մեջ [webpack](https://webpack.js.org/)-ի հզորությունը և preset-ների պարզությունը. ներառում է preset [React հավելվածների](https://neutrinojs.org/packages/react/) և [React կոմպոնենտների](https://neutrinojs.org/packages/react-components/) համար։
```

Comment:
"and includes" -> "և ներառում է"։ Միջակետ դնելով, դու տպավորություն ես ստեղծում, թե համատեղելն ա բացատրվում էս ներառելով, ինչը ենթադրում եմ սխալ ա։